### PR TITLE
Stable/1.0.4 4

### PR DESCRIPTION
**Pull request by munroesj52**

### CODE REVIEW

**Comment by munroesj52** [comment]
I really did sync munroesj52 with upstream. Don't understand where this conflict is coming from. But I can resolve this and merge.

Wanted to give you a chance to review.

File: src/testsuite/arith128_test_i128.c [hunk 1]
```diff
@@ -2308,7 +2308,8 @@ test_msumudm (void)
 int
 test_muludq (void)
 {
-  vui32_t i, j, k, l /*, m*/;
+  vui32_t i, j, k/*, l , m*/;
```
Comment (ThinkOpenly) [inline src/testsuite/arith128_test_i128.c:2311]: Do you want to keep the obsolete code in the comment?

**Review by ThinkOpenly — CHANGES_REQUESTED** [review]
In the 1st commit, I think it'd be nice to describe the compiler errors you are rectifying.

In the 2nd commit message:
> The implementation (tries again) to supports.

s/(tries again)/tries (again)/
s/supports/support/

>	* src/pveclib/vec_f128_ppc.h (__binary128): Define/typedef to
>	the quad-precision float type that the support supports.

s/support supports/compiler supports/ ?

File: src/testsuite/arith128_test_i128.c (hunk 1 above)
Comment (munroesj52) [reply src/testsuite/arith128_test_i128.c:2311]: Leave it for now. Fix it in upstream.

**Comment by munroesj52** [comment]
Need to merge this for testing

**Pull request merged**